sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -77,6 +80,34 @@ public EditEventSteps(WebDriverHelpers webDriverHelpers, Faker faker) {
               .isEqualTo(CreateNewEventSteps.newEvent.getEventLocation());
         });
 
+    When(
+        "I change the fields of event and save",
+        () -> {
+          event = eventService.buildEditEvent();
+          fillDateOfReport(event.getReportDate());
+          fillStartData(event.getEventDate());
+          event =
+              event.toBuilder().uuid(webDriverHelpers.getValueFromWebElement(UUID_INPUT)).build();
+          selectEventStatus(event.getEventStatus());
+          selectEventInvestigationStatusOptions(event.getInvestigationStatus());
+          selectEventManagementStatusOption(event.getEventManagementStatus());
+          selectRiskLevel(event.getRiskLevel());
+          selectDisease(event.getDisease());
+          fillTitle(event.getTitle());
+          selectSourceType(event.getSourceType());
+          selectTypeOfPlace(event.getEventLocation());
+          webDriverHelpers.scrollToElement(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_DATA_SAVED_MESSAGE);
+        });
+
+    When(
+        "I check the modified event data is correctly displayed",
+        () -> {
+          final Event currentEvent = collectEventData();
```

Comment:
we can remove the final keyword. As I've seen in other tests, sometimes we can update created/updated/collected POJOs from steps, and use them in other tests :D